doc/sphinx/language/core/definitions.rst

```diff
@@ -92,30 +106,26 @@ Section :ref:`typing-rules`.
       | {* @binder } : @type
       reduce ::= Eval @red_expr in
 
-   These commands bind :n:`@term` to the name :n:`@ident` in the global environment,
-   provided that :n:`@term` is well-typed.  They can take the :attr:`local` :term:`attribute`,
-   which makes the defined :n:`@ident` accessible only through their fully
-   qualified names, even if :cmd:`Import` or its variants has been used on the
-   current :cmd:`Module`.
+   This binds :n:`@term` to the name :n:`@ident` in the global environment,
+   provided that :n:`@term` is well-typed.
+
+   If :n:`@type` is specified, the command checks that the type of :n:`@term`
+   is definitionally equal to :n:`@type`.
+
+   If :n:`@binder` is specified, it distributes over :n:`@term` and :n:`@type` as if they had
+   respectively been :n:`fun {* @binder } => @term` and :n:`forall {* @binder }, @type`.
```

Comment:
```suggestion
   If :n:`@binder` is specified, :n:`@term` and :n:`@type` are treated as if they had
   respectively been given as :n:`fun {* @binder } => @term` and :n:`forall {* @binder }, @type`.
```

"distributes" suggest to me a reference to a distributive property, which IINM is not the case--it just has it's everyday meaning.